.github/workflows/generate-pr-description.yml

```diff
@@ -0,0 +1,61 @@
+name: Generate PR Description
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  generate-pr-description:
+    permissions:
+      pull-requests: write
+      statuses: write
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/sk generate-pr-description')
+    steps:
+      - name: Get PR branch
+        uses: xt0rted/pull-request-comment-branch@v1
+        id: comment-branch
+
+      - name: Set latest commit status as pending
+        uses: myrotvorets/set-commit-status-action@master
+        with:
+          sha: ${{ steps.comment-branch.outputs.head_sha }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          status: pending
+
+      - name: Generate PR description
+        uses: mkarle/skonsole-generate-pr-description@v1
+        with:
+          pull-request-number: ${{ github.event.issue.number }}
+          pull-request-diff-url: ${{ github.event.issue.pull_request.diff_url }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          update-type:  ${{ contains(github.event.comment.body, 'replace') && 'replace' || (contains(github.event.comment.body, 'prefix') && 'prefix' || 'suffix') }}
+        env: # Set Azure credentials secret as an input
+          AZURE_OPENAI_CHAT_DEPLOYMENT_NAME: ${{ vars.AZUREOPENAI__CHAT__DEPLOYMENTNAME }}
```

Comment:
Perfect for now -- we may want a separate variable name if we want toggle it without impacting other actions.